src/adminserver/systemcfg/systemcfg.go

```diff
@@ -363,7 +372,18 @@ func LoadFromEnv(cfgs map[string]interface{}, all bool) error {
 		}
 	}
 
+	reloadCfg := os.Getenv("RESET")
+	skipReg := os.Getenv("SKIP_RELOAD_ENV_REG")
+	skipReg = strings.TrimSpace(skipReg)
+	if len(skipReg) == 0 {
+		skipReg = "$^" // doesn't match any string by default
+	}
+	skipMatcher := regexp.MustCompile(skipReg)
```

Comment:
This maybe too strict that if user input a wrong pattern it will panic, how about if it fails to compile the pattern match will not work?

Reply:
Updated